README.md

```diff
@@ -2010,6 +2010,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [R Rushikesh Gandhmal](https://github.com/RushikeshGandhmal)
   - [R Varshith Kumar](https://github.com/codefromrvk)
   - [Rachita Goyal](https://github.com/rachita2000)
+  - [Rachael Odetayo](https://github.com/Rakielle)
```

Comment:
Please keep the list in the correct alphabetical order.
```suggestion
  - [Rachael Odetayo](https://github.com/Rakielle)
  - [Rachita Goyal](https://github.com/rachita2000)
```

Reply:
Done. Please review it.